test/Evaluate/folding01.f90

```diff
@@ -1,3 +1,4 @@
+!RUN: %S/test_folding.sh %s %flang
```

Comment:
Might it be worth adding "tools" for these in lit.cfg.py so instead of doing %S/test_folding.sh we can do %folding? It might just be a little neater than having to specify the path to the script.

Reply:
Funny you should say that! I did this originally, but both @tskeith and @RichBarton-Arm said they prefer it like this (I think it was something to do with the neatness of the config file being preferable).

Reply:
Yes, I still prefer it like this.

Reply:
Happy to go with the majority opinion on this one, disregard my comment